tests/op/extract/load/test_html_op.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from unittest.mock import patch
+
+from uniflow.node import Node
+from uniflow.op.extract.load.html_op import ExtractHTMLOp, ProcessHTMLOp
+
+
+class TestExtractHTMLOp(unittest.TestCase):
+    def test_extract_html_op_with_url(self):
+        extract_op = ExtractHTMLOp("test_extract")
```

Comment:
shall we init the function in the class?
e.g. `self.extract_op = ExtractHTMLOp("test_splitter")`